init-pants/action.yaml

```diff
@@ -66,19 +85,19 @@ runs:
       shell: bash
       run: |
         if ! command -v pants; then
+          echo "$HOME/bin" >> $GITHUB_PATH
```

Comment:
I moved this to the top to avoid warnings from the pants setup script that `$HOME/bin` is not on the `$PATH`.

Reply:
Noting that this won't have that effect. `echo "$HOME/bin" >> $GITHUB_PATH` causes that dir to be prepended to `$PATH` starting from the next action. It does not change this action's `$PATH`. So that warning will still be there. E.g., see [here](https://github.com/pantsbuild/example-python/actions/runs/6039726248/job/16389029547)

Reply:
I know @benjy and it makes me so sad 😢

Reply:
But this sadness could be mitigated with an additional `echo "$HOME/bin" >> $PATH`, no?

Reply:
Not sad enough to fix it 🤪😁